docs/rules/no-import-of-scoped-modules-during-ssr.md

```diff
@@ -0,0 +1,72 @@
+# Disallow static import of scoped modules during SSR (`lwc/no-import-of-scoped-modules-during-ssr`)
+
+During server-side rendering (SSR), these modules resolve to static values on the server and are updated to actual values on the client.
+
+## Rule details
+
+**Static imports of user-scoped modules:**
+
+Example of **incorrect** code:
+
+```js
+import { LightningElement } from 'lwc';
+import userId from '@salesforce/user/Id';
+import isGuest from '@salesforce/user/isGuest';
+
+export default class Foo extends LightningElement {
+    // This component should not import user-scoped modules during SSR
+}
+```
+
+Example of **correct** code:
+
+**Dynamic imports of user-scoped modules:**
+
+Example of **incorrect** code:
+
+```js
+import { LightningElement } from 'lwc';
+
+export default class Foo extends LightningElement {
+    connectedCallback() {
+        import('@salesforce/userPermission').then((module) => {
+           ...
+        });
+    }
+}
+```
+
+Example of **correct** code:
+
+**Imports of formFactor-scoped module:**
+
+Example of **incorrect** code:
+
+```js
+import { LightningElement } from 'lwc';
+import formFactor from '@salesforce/client/formFactor';
+
+export default class Foo extends LightningElement {
+   ...
+}
+```
+
+Example of **correct** code:
+
+Render all form factor outputs and show/hide with media queries https://jsbin.com/gasuyidixa/edit?html,css,output.
```

Comment:
Linking to a third party host we don't control makes this doc more fragile than eg inlining the example. Please revise.

Reply:
The public doc hasn't been updated to show a form factor example for 254 yet. Eventually, it will be here: https://developer.salesforce.com/docs/platform/lwr/guide/lwr-portable-best-practices.html

I'm not sure what to do in the meantime.